ScaleformUI_Lua/src/scaleforms/BigMessageScaleform/BigMessageInstance.lua

```diff
@@ -57,7 +57,7 @@ function m:ShowSimpleShard(msg, subtitle, time)
     if time == nil then  time = 5000 end
     self:Load()
     self._start = GetGameTimer()
-    self._sc:CallFunction("SHOW_SHARD_CREW_RANKUP_MP_MESSAGE", false, title, subtitle)
```

Comment:
the error here is that msg should be called title.. and title is not nil in the scaleform

Reply:
i change it